arch/sparc/src/sparc_v8/up_initialstate.c

```diff
@@ -71,7 +71,7 @@ void up_initial_state(struct tcb_s *tcb)
   if (tcb->pid == IDLE_PROCESS_ID)
     {
       tcb->stack_alloc_ptr = (void *)(g_idle_topstack -
-                                      (CONFIG_SMP_NCPUS * CONFIG_IDLETHREAD_STACKSIZE));
+                           (CONFIG_SMP_NCPUS * CONFIG_IDLETHREAD_STACKSIZE));
```

Comment:
why change?

Reply:
ci long line error

Reply:
ok, should we move to the first patch?

Reply:
i don't  proficient in github,it could be a mess

Reply:
it's simple:
git rebase --interactive HEAD~2
change pick to edit for the first patch
make the change and git commit --amend
git rebase --continue

Reply:
it means i should fix->commit->rebase->squash->amend file one by one, rather than fix all file and the rebase?